### PR TITLE
Convert Affymetrix.Rnw to Affymetrix.Rmd

### PR DESCRIPTION
**Pull request by sonali8434**
@smgogarten kindly review this PR. We use this [Review PR](https://bioconductor.github.io/sweave2rmd/articles/contribute.html#review-a-pull-request) section of our document to review it. If you want us to make any changes in .Rmd file then please let us know. Any suggestions would be appreciated.

After you merge the PR, remember to bump the version in the DESCRIPTION file so that the Bioconductor build system will recognize the new changes and [push the changes to Bioconductor's git repository](https://contributions.bioconductor.org/git-version-control.html#push-to-github-bioc).

Attached: 

Original [PDF](https://bioconductor.org/packages/release/bioc/vignettes/GWASTools/inst/doc/Affymetrix.pdf)
Converted [HTML](https://rpubs.com/Sonali8434/1054966)

### CODE REVIEW

File: vignettes/Affymetrix.Rmd
Comment (smgogarten) [inline vignettes/Affymetrix.Rmd]: Please change to:
name: "GENEVA Coordinating Center"
affiliation: "Department of Biostatistics, University of Washington"

**Review by smgogarten — CHANGES_REQUESTED** [review]
I noted one change about the author/affiliation split, otherwise this looks great. Thank you!

**Comment by sonali8434** [comment]
Hello @smgogarten, I have made the suggested changes.

 I want to inform you about one more thing, and I apologize for not mentioning it earlier. Generally, we give credits to the converter for converting Sweave documents. We ask maintainers if they want us to add ourselves as contributors in the description file and the .Rmd file. I wanted to ask if you would like me to add as a contributor for this conversion. It's totally fine to say no.

 Thank you so much for your time.

**Review by smgogarten — APPROVED** [review]

**Comment by smgogarten** [comment]
Yes, go ahead and add yourself as a contributor. Thanks!

**Comment by sonali8434** [comment]
Thank you! 

After you merge the PR, remember to bump the version in the DESCRIPTION file so that the Bioconductor build system will recognize the new changes and [push the changes to Bioconductor's git repository](https://contributions.bioconductor.org/git-version-control.html#push-to-github-bioc).

**Pull request merged**
